crates/evm/src/evm/mod.rs

```diff
@@ -89,7 +89,9 @@ impl StateValueCodec<AccountInfo> for BorshCodec {
     type Error = std::io::Error;
 
     fn encode_value(&self, value: &AccountInfo) -> Vec<u8> {
-        borsh::to_vec(&value).unwrap()
+        let mut buf = Vec::with_capacity(32 + 8 + 32 + 1);
```

Comment:
do you think it would be better if we check code_hash.is_some and add the last +32 accordingly? It is also possible that compiler might make better optimizations if the capacity is constant, and extra +32 bytes almost never gonna mean much.

Reply:
Unfortunately we go into Allocator which doesn't care much about that difference. We need to bench this case but I doubt it will matter at all

Reply:
then lets keep it this way, its better than 1024 in any case